fastqheat/__main__.py

```diff
@@ -48,21 +49,35 @@ def get_program_version(program_name: str) -> tp.Optional[str]:
         return output
 
 
-def _make_accession_list(term: str) -> list[str]:
+def _make_accession_list(terms: tp.Iterable[str]) -> list[str]:
     """Get an accession list based on pattern of the given term."""
-    if SRR_PATTERN.search(term):
-        accession_list = [term]
-    elif SRP_PATTERN.search(term):
-        accession_list = ENAClient().get_srr_ids_from_srp(term)
-    else:
-        raise ValueError(f"Unknown pattern: {term}")
-
+    accession_list = list()
+    for term in terms:
+        if not term:
```

Comment:
Is it possible that term can be falsy? if so, how?

Reply:
it is not ideal solution, but there can be empty stings in input file. Since file may be quite large it looks like a good place to handle this

Reply:
@dmitriy-grossman-quantori thank you, now I understand. However, if it only concerns file input, maybe it's more appropriate to get rid of empty strings within `validate_accession_file()`?

Reply:
That was my initial thought, but it looks even worse or consumes memory unnecessary.

Reply:
@dmitriy-grossman-quantori okay. Can you please leave a comment about it here in this case? Something like 'there can be empty stings in input file, for now we decided to handle it here because doing it in `validate_accession_file()` looks even worse or consumes memory unnecessary.'